hub/configs/core/horizon/hzn.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think I understand why you are adding all of these files to source control, but usually when we do that, it's a good idea to provide defaults for this file so that folks can just run the example out of the repo. Can we do that in this case? if not, then the value of source controlling these files is somewhat limited. In that case, maybe only source control the example service def file?